From 2776beeaaaa0a759d4d8ed9b09b5047b4c158c3c Mon Sep 17 00:00:00 2001
From: Jose Dapena Paz <jdapena@igalia.com>
Date: Tue, 22 Sep 2020 19:08:23 +0000
Subject: [PATCH] IWYU: std::sort requires include algorithm in AutoFillOfferData.

Fixes this compilation error:
../../components/autofill/core/browser/data_model/autofill_offer_data.cc:56:8: error: ‘sort’ is not a member of ‘std’; did you mean ‘sqrt’?

Bug: 957519
Change-Id: Iaef429cfca5e55af8b819457f9b9729ab4f669f5
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/2423586
Commit-Queue: Parastoo Geranmayeh <parastoog@google.com>
Reviewed-by: Parastoo Geranmayeh <parastoog@google.com>
Cr-Commit-Position: refs/heads/master@{#809447}
---

diff --git a/components/autofill/core/browser/data_model/autofill_offer_data.cc b/components/autofill/core/browser/data_model/autofill_offer_data.cc
index 752e21e..228482b 100644
--- a/components/autofill/core/browser/data_model/autofill_offer_data.cc
+++ b/components/autofill/core/browser/data_model/autofill_offer_data.cc
@@ -4,6 +4,8 @@
 
 #include "components/autofill/core/browser/data_model/autofill_offer_data.h"
 
+#include <algorithm>
+
 namespace autofill {
 
 AutofillOfferData::AutofillOfferData() = default;
@@ -65,4 +67,4 @@
   return 0;
 }
 
-}  // namespace autofill
\ No newline at end of file
+}  // namespace autofill
